core/domain/exp_services.py

```diff
@@ -1925,10 +1928,21 @@ def are_changes_mergeable(exp_id, frontend_version, change_list):
             value: key for key, value in new_to_old_state_names.items()
         }
 
+        change_list_dict = [change.to_dict() for change in change_list]
         if len(added_state_names) > 0 or len(deleted_state_names) > 0:
-            # Here we will send the changelist, version, latest_version,
-            # and exploration to the admin, so that the conditions
-            # can be reviewed.
+            # In case of the addition and the deletion of the state,
+            # we are rejecting the mergebility because these cases
+            # change the flow of the exploration and are quite complex
+            # for now to handle. So in such cases, we are sending the
+            # changelist, frontend_version, backend_version and
+            # exploration id to the admin, so that we can look into the
+            # situations and can figure out the way if it’s possible to
+            # handle these cases.
+            if feconf.CAN_SEND_EMAILS:
+                email_manager.send_not_mergeable_change_list_to_admin_for_review_email( # pylint: disable=line-too-long
```

Comment:
```suggestion
                (
                     email_manager
                     .send_not_mergeable_change_list_to_admin_for_review_email( 
```

Can we do this^? (And avoid having bypass flag)

Reply:
Done.